Gemfile

```diff
@@ -4,12 +4,13 @@ gemspec
 
 group :development, :test do
   gem 'between_meals', :git => 'https://github.com/facebook/between-meals'
-  gem 'chef-dk', '~> 3.13.0'
-  gem 'chef-zero'
+  gem 'chef', '= 16.18.0'
```

Comment:
this should be chef-workstation these days.

Reply:
OK, so this is tricky.

*TECHNICALLY* GD, BM, and TT don't have a direct dependency on dk/ws.

However, they're generally intended to be installed as a devtools along side dk/ws. And we try to move more and more stuff into WS so we don't have to manage it in a fb-chefdk-gems sorta way. That's true, for example, with markdownlint, we got that into WS so we could stop managing it ourselves.

You're right though that strictly speaking, we don't need ws/dk, we just use those to provide our actual deps.